### DISCUSSION TITLE
Update CDN enforced IP automatically on user side تغییر خودکار آیپی اینفورس شده CDN در کانفیگهای کاربر

### DISCUSSION THREAD

**Discussion by lymanjre** [hiddify/Hiddify-Manager#144] [Ideas ایدهها]
I just wanna say that is it possible to update enforced IP of CDN into user configs? I mean, after user adds the config into the application, the Inspector may block that CDN IP; then we change that IP with a new one. Is it possible on the user side it wouldn't need any changes?

آیا امکان داره وقتی آیپی اینفورس شده cdn سمت سرور تغییر میکنه؛ این تغییر آیپی سمت کاربر به صورت خودکار انجام بشه؟

**Comment by ghost** [comment]
در نرم نرم افزار کلش و Hiddify چون شما یه لینک رو subscribe کردی اگر همون رو رفرش کنید تغییرات اعمال می شه در کلاینت مگر اینکه دامین رو تغییر داده باشی کلا یا uiid  یوز  تفاوت کرده باشه مثلا حذف کردین دوباره add کردین
قابلیت scheduling هم وجود داره

**Comment by raminrez** [comment]
برای رفع این مشکل به‌جای اینکه آپی رو مستقیم از داخل پنل تغییر بدید بهتره که DNS query ست کرده باشید و از اونجا تغییر بدید٬ اینجوری حتی وقتی آی‌پی ها عوض میشن کاربر نیاز نداره پروفایلش رو آپدیت کنه

لینک آموزش اینکار که سگارو زحمتشو کشیده از توویتر میزارم برات

[https://twitter.com/iSegaro/status/1625919471461203971](https://twitter.com/iSegaro/status/1625919471461203971)
